### DISCUSSION TITLE
"Updates" dashboard: Add 'You are on x% rollout' column

### DISCUSSION THREAD

**Discussion by KrzysztofHajdamowicz** [teslamate-org/teslamate#3791] [Ideas]
When visiting webpage https://www.notateslaapp.com/software-updates/version/2024.8.7/statistics you can see % of cars (in that fleet) that got specific software version.
Teslamate can leverage that and when car receives software update, request can be made and "At which percentile I am?" logged into database.
This can help in tracking how early in deployment rollout was the car across its lifetime.

**Comment by KrzysztofHajdamowicz** [comment]
Looks like there might be quite usable data source for that:
https://stats.tessie.com/

**Reply by KrzysztofHajdamowicz** [reply]
@Bre77 would you be OK if Teslamate scrape stats webpage when firmware upgrade is available for car?

**Reply by parkr** [reply]
On https://www.notateslaapp.com/software-updates/, I see a reference to TeslaFi. https://teslafi.com/firmware

**Reply by Bre77** [reply]
@KrzysztofHajdamowicz I don't represent Tessie, so you'll have to reach out to James by email
